src/rockstor/storageadmin/static/storageadmin/js/views/dashboard/pool_usage.js

```diff
@@ -45,17 +48,23 @@ PoolUsageWidget = RockStorWidgetView.extend({
         }));
         this.pools.fetch({
             success: function(collection, response, options) {
+
                 _this.setData();
                 _this.setGraphDimensions();
                 _this.renderPools();
-        }
+            }
         });
+
         return this;
     },
 
     setData: function() {
-        this.data = this.pools.sortBy(function(p) { return p.get('name'); }).slice(0, this.numTop);
-        this.data.map(function(d) {
+
+        var _this = this;
+        _this.data = this.pools.sortBy(function(p) {
```

Comment:
Really appreciate this improvement! The one line functions kill me and this code hasn't had other eyes for a while 😄